ShchyrbaOleksandr/HW6/task1.py

```diff
@@ -0,0 +1,19 @@
+main_list = range(1, 11)
```

Comment:
good job
look at this code-more optimized version of the code

main_list = range(1, 11)

even_numbers_2 = []
odd_numbers_3 = []
numbers_2_3 = []

for num in main_list:
    if num % 2 == 0:
        even_numbers_2.append(num)
    elif num % 2 != 0 and num % 3 == 0:
        odd_numbers_3.append(num)
    elif num % 2 != 0 and num % 3 != 0:
        numbers_2_3.append(num)

print(even_numbers_2)
print(odd_numbers_3)
print(numbers_2_3)